server/graphql/types/src/types/vaccine_course_item.rs

```diff
@@ -3,42 +3,44 @@ use dataloader::DataLoader;
 
 use graphql_core::{loader::ItemLoader, standard_graphql_error::StandardGraphqlError, ContextExt};
 
-use repository::vaccine_course::vaccine_course_item_row::VaccineCourseItemRow;
+use repository::vaccine_course::vaccine_course_item::VaccineCourseItem;
 
 use super::ItemNode;
 
 #[derive(PartialEq, Debug)]
 pub struct VaccineCourseItemNode {
-    pub vaccine_course_item: VaccineCourseItemRow,
+    pub vaccine_course_item: VaccineCourseItem,
 }
 
 #[Object]
 impl VaccineCourseItemNode {
     pub async fn id(&self) -> &str {
-        &self.row().id
+        &self.row().vaccine_course_item.id
     }
 
     pub async fn item(&self, ctx: &Context<'_>) -> Result<ItemNode> {
         let loader = ctx.get_loader::<DataLoader<ItemLoader>>();
-        let item_option = loader.load_one(self.row().item_link_id.clone()).await?;
+        let item_option = loader.load_one(self.row().item.id.clone()).await?;
 
         let item = item_option.ok_or(
-            StandardGraphqlError::InternalError(format!("Cannot find item {}", self.row().id,))
-                .extend(),
+            StandardGraphqlError::InternalError(
+                format!("Cannot find item {}", self.row().item.id,),
+            )
+            .extend(),
         )?;
 
         Ok(ItemNode::from_domain(item))
     }
 }
 
 impl VaccineCourseItemNode {
-    pub fn from_domain(vaccine_course_item: VaccineCourseItemRow) -> VaccineCourseItemNode {
+    pub fn from_domain(vaccine_course_item: VaccineCourseItem) -> VaccineCourseItemNode {
         VaccineCourseItemNode {
             vaccine_course_item,
         }
     }
 
-    pub fn row(&self) -> &VaccineCourseItemRow {
+    pub fn row(&self) -> &VaccineCourseItem {
```

Comment:
Is this case it probalby doesn't matter, but I think part of the point of the `row()` call is to avoid needing to update all the places when something like this changes.
So  you could do 
 pub fn row(&self) -> &VaccineCourseItemRow {
        &self.vaccine_course_item.vaccine_course_item
    }
pub fn item_row(&self) -> ItemRow {
        &self.vaccine_course_item.item
}